ddtrace/contrib/futures/threading.py

```diff
@@ -17,9 +20,10 @@ def _wrap_submit(func, args, kwargs):
     #
     #      The resolution is to not create/propagate a new context if one does not exist, but let the
     #      future's thread create the context instead.
-    current_ctx = None
+    # DEV: Be sure to propagate a Context and not a Span since we are crossing thread boundaries
+    current_ctx: Optional[Context] = None
     if ddtrace.tracer.context_provider._has_active_context():
```

Comment:
We can remove this line. We don't need to check this internal api. If the tracer does not have an active context then the suggestion above will return None.